packages/testlab/index.ts

```diff
@@ -3,5 +3,6 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-// NOTE(bajtos) This file is used by VSCode/TypeScriptServer at dev time only
+// DO NOT EDIT THIS FILE
+// Add any aditional (re)exports to src/index.ts instead.
 export * from './src/testlab';
```

Comment:
Not a blocker. Is there a reason in `@loopback/testlab` we use `testlab.ts` as the top level file instead of `index.ts`?

Reply:
I personally dislike to have so many copies of `index.ts` around, because one can get quickly confused about which index file he/she is editing.

Historically, `testlab` started small with `should` and `sinon` only, so there was only a single file and it made sense to call it `testlab.ts`

That's no longer the case, and since other packages are using `index.ts` all over the place, I agree we should change testlab to be consistent with the rest of the monorepo. I made that change in the now-discarded commit  https://github.com/strongloop/loopback-next/commit/08a6a3439ebc6f8de5a6e8bc148824076ac33e83, I'll extract it back to this patch.